pypeit/spectrographs/keck_deimos.py

```diff
@@ -286,6 +336,11 @@ def config_specific_par(self, scifile, inp_par=None):
         # Arc lamps list from header
         par['calibrations']['wavelengths']['lamps'] = ['use_header']
 
+        # increase order of final fit because better for mosaic (mosaic is the default)
+        par['calibrations']['wavelengths']['n_final'] = 6
```

Comment:
Is it possible to know here whether we are in mosaic mode and act accordingly?

same for `sigdetect` below..